felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/rapidsrivers/StatefullDataKanal.kt

```diff
@@ -62,7 +62,9 @@ class StatefullDataKanal(
     }
 
     fun isAllDataCollected(key: RedisKey): Boolean {
-        return redisStore.exist(*dataFelter.map { key.toString() + it }.toTypedArray()) == dataFelter.size.toLong()
+        val numKeysInRedis = redisStore.exist(*dataFelter.map { key.toString() + it }.toTypedArray())
+        println("found " + numKeysInRedis)
```

Comment:
kan man bruke logger istedenfor println her?

Reply:
Ja alle disse fjerner jeg. Hadde ikke tid for det før PC-en min døde.